task.go

```diff
@@ -232,13 +233,13 @@ type TaskUpdate struct {
 
 	// LatestCompleted us to set latest completed on startup to skip task catchup
 	LatestCompleted *string                `json:"-"`
+	LatestScheduled *time.Time             `json:"-"`
```

Comment:
no need for this to be a pointer, it simplifies checking if you make it a time.Time

Reply:
This is the TaskUpdate struct. I believe we want a pointer so we can pass nil latestScheduled and not apply an update.